packages/svelte-ux/src/lib/components/ThemeSwitch.svelte

```diff
@@ -16,22 +25,34 @@
     let newTheme = e.target?.checked ? 'dark' : 'light';
     currentTheme.setTheme(newTheme);
   }}
-  classes={{
-    switch: 'dark:bg-primary dark:border-primary',
-    toggle: 'translate-x-0 dark:translate-x-full',
-  }}
+  classes={clsMerge(
+    {
+      switch: 'dark:bg-primary dark:border-primary',
+      toggle: 'translate-x-0 dark:translate-x-full',
+    },
+    settingsClasses.root,
+    classes
+  )}
   {...$$restProps}
 >
   <div class="grid grid-cols-1 grid-rows-1">
     <Icon
       data={mdiWeatherNight}
       size=".8rem"
-      class="row-[1] col-[1] text-primary opacity-0 dark:opacity-100"
+      class={cls(
+        'row-[1] col-[1] text-primary opacity-0 dark:opacity-100',
+        settingsClasses.icon,
```

Comment:
is it correct that this should inherit the `ThemeSwitch`-specific icon classes or should it instead be the global `Icon` classes? Or a combo of the two? 🤔

Reply:
it should be the `ThemeSwitch` icon specific case here, but `Icon` settings will also apply "globally" to all `Icon` instances (direct or within other components such as here within ThemeSwitch).